stumpy/core.py

```diff
@@ -4289,3 +4289,82 @@ def process_isconstant(T, m, T_subseq_isconstant, T_subseq_isfinite=None):
     )
 
     return T_subseq_isconstant
+
+
+def deco_ray_tor(f):
+    """
+    Wraps a `numba` JIT-compiled function as a Python function.
+
+    This "indirection" is required for Ray serialization to work.
+
+    Parameters
+    ----------
+    f : function
+        A `numba` JIT-compiled function
+
+    Returns
+    -------
+    wrapper : function
+        A Python function
+    """
+
+    def wrapper(*args):  # pragma: no cover
+        return f(*args)
```

Comment:
I considered it but I don't want to allow a larger attack surface for things to go wrong. This should be sufficient

Reply:
No, I don't think it's worth it. I'm trying to do as little as possible here and adding `**kwargs` makes it seem like we are allowing "more" and I really want to restrict this function. The only reason this stupid wrapper is needed is because `ray` needs to serialize the `numba` function but the `numba` function has a signature that is unrecognized by `ray` as "python function" so this is the hack. I really hate this enough already and so I don't want to add more stuff to it to allow it to break

Reply:
>  I'm trying to do as little as possible here

Got the perspective. Thanks for the explanation!